### DISCUSSION TITLE
Which text boxes are impossible for teachers to view.

### DISCUSSION THREAD

**Discussion by crazyguy1908** [SafeExamBrowser/seb-win-refactoring#610] [Q&A]
I was wondering which text boxes could children create inside SEB that would be impossible for us to view as they were giving thier exam and how we could prevent them from using them. Is there any way for a student to create a textbox we cannot view?

**Comment by dbuechel** [comment]
No, not if you configure SEB correctly and if your students do not have a way to manipulate the HTML page content itself.

**Reply by crazyguy1908** [reply]
I myself have identified the cntrl F popup and the note function on the graphing calculator as 2 possibilities.
